ifrs17/Test/TestData.ipynb

```diff
@@ -15,6 +15,15 @@
     "nbformat": 4,
     "nbformat_minor": 5,
     "cells": [
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"../CalculationEngine\""
```

Comment:
I would import only what is necessary, in this way we can keep compilation time down.

Reply:
ok agree